.github/workflows/e2e-linux.yml

```diff
@@ -32,6 +32,14 @@ jobs:
   test:
     runs-on: ubuntu-latest
 
+    env:
```

Comment:
Let's remove all changes to `.github/workflows/e2e-linux.yml` :pray: . I'll handle it under https://github.com/input-output-hk/cardano-wallet/pull/3643

Reply:
👌🏼